webapp/app.py

```diff
@@ -100,9 +110,49 @@ def api():
     Show the static api page
     """
 
+    base_url = "https://raw.githubusercontent.com/maas/maas-openapi-yaml/main"
```

Comment:
If I were you I'd just put the full URL in one variable, and the local filename in another unrelated variable. There's no need for them to be the same filename.

Reply:
Honestly the only reason I did it this way was to get around linting errors. The max line length in our checks is 79, and setting the whole URL to a variable exceeded that length. Bit of a hacky workaround but I'm very open to suggestions for better implementations for this, been a while since I've used Python so I'm a little rusty